classes/PodsUI.php

```diff
@@ -3058,7 +3108,18 @@ public function manage( $reorder = false ) {
 									<?php esc_html_e( $filter_field['label'] ); ?>
 									</label>
 									<?php
-									echo PodsForm::field( 'filter_' . $filter, $value, 'pick', $options );
+									// Prevent p div issues.
+									echo str_replace(
+										array(
+											'<div',
+											'</div>',
+										),
+										array(
+											'<span',
+											'</span>',
+										),
+										PodsForm::field( 'filter_' . $filter, $value, 'pick', $options )
+									);
```

Comment:
:warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: ffd3e4dc890d5b9e228d13da0a6011bb</sub>